cl/favorites/urls.py

```diff
@@ -22,4 +23,5 @@
         name="view_tag",
     ),
     path("tags/<str:username>/", view_tags, name="tag_list"),
+    path("recap_requests/", open_prayers, name="recap_requests"),
```

Comment:
We'll want this URL to be better. This is the leader board of the most wanted docs in PACER. Perhaps:

`/prayers/top/`

And we could change the function name to `top_prayers`, and the template name to `top_prayers.html` as well?

Reply:
Done in #[8a36f56](https://github.com/freelawproject/courtlistener/pull/4386/commits/8a36f56065f3a4587b35ddec11c3e6bcce39f178)